src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -0,0 +1,340 @@
+﻿using Mono.Cecil;
+using Mono.Cecil.Cil;
+using System;
+using System.Collections.Generic;
+
+namespace Mono.Linker.Dataflow
+{
+	// TODO: enforce virtual methods have consistent annotations
```

Comment:
File a tracking issue for this - just in case :-)

Reply:
#1028